solution/build.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/bash
+set -e
+set -u
+
+PWD=$(pwd)
+BUILD_ROOT=${PWD}/_build
+ISO_ROOT=${BUILD_ROOT}/root
+IMAGES_ROOT=${ISO_ROOT}/images
+
+PRODUCT_NAME=Zenko
+PRODUCT_LOWERNAME=zenko
+BUILD_TIMESTAMP=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
+BUILD_HOST=$(hostname)
+
+VERSION_SHORT=$(git describe --abbrev=0)
+VERSION_FULL=${VERSION_SHORT}-dev
+GIT_REVISION=$(git describe --long --always --tags --dirty)
+ISO=${BUILD_ROOT}/${PRODUCT_LOWERNAME}-${VERSION_FULL}.iso
+
+DOCKER=docker
+DOCKER_OPTS=
+DOCKER_SOCKET=unix:///var/run/docker.sock
+HARDLINK=hardlink
+OPERATOR_SDK=operator-sdk
+OPERATOR_SDK_OPTS=
+SKOPEO=skopeo
+SKOPEO_OPTS="--override-os linux --insecure-policy"
+OPERATOR_TAG=$(grep /zenko-operator: deps.txt | awk -F ':' '{print $2}')
+
+# grab our dependencies from our deps.txt file as an array
+readarray -t DEP_IMAGES < deps.txt
+
+function clean()
+{
+	rm -rf ${BUILD_ROOT}
+}
+
+function mkdirs()
+{
+	mkdir -p ${ISO_ROOT}
+}
+
+function gen_manifest_yaml()
+{
+	cat > ${ISO_ROOT}/manifest.yaml <<EOF
+apiVersion: solutions.metalk8s.scality.com/v1alpha1
+kind: SolutionConfig
```

Comment:
The kind is now `Solution`:
```suggestion
kind: Solution
```